README.md

```diff
@@ -43,11 +43,11 @@
 
 ## Overview
 
-* **Modern features brought to IRC.** Push notifications, link previews, new message markers, and more bring IRC to the 21st century.
-* **Always connected.** Remains connected to IRC servers while you are offline.
-* **Cross platform.** It doesn't matter what OS you use, it just works wherever Node.js runs.
-* **Responsive interface.** The client works smoothly on every desktop, smartphone and tablet.
-* **Synchronized experience.** Always resume where you left off no matter what device.
+- **Modern features brought to IRC.** Push notifications, link previews, new message markers, and more bring IRC to the 21st century.
```

Comment:
Why is it replacing our bullets with `-`?

Reply:
Because that's also fairly accepted to use `-` for bullets rather than `*` in Markdown (note that the outcome in generated HTML is the same, HTML `li`s). One reason is that `*` is also used for bold (and sometimes italics), and that becomes confusing to read: `* **` is a bullet followed by bold (like it was here), `***` is italics+bold`, etc.

I just noticed that GitHub also uses `-` when using the "Add a bulleted list" button in the comment input.

Also this was the only place in Markdown files in this repo where we're using `*` (and we don't on the website), so that's not so disruptive.